code/modules/clothing/head/misc.dm

```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/head/canada
 	name = "striped red tophat"
-	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'�rable</i>"
+	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'�rable</i>"

```

Comment:
didnt even touch it, this keeps happening in Atom whenever I touch a part of code with shitty text stuff like that.